community/tools/json2graphql/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing to JSON Data Import
+
+## Issues
+
+Please open an issue related to your work. Add the label `c/json2graphql`.
+
+## Local developmet
+
+1. Make changes and save
+2. Run the executable in the `bin` directory to test your code. Treat the executable as the command. For example:
+
+   ```
+   $ bin/run --help
+   ```
+
+## Testing
+
+No pull request will be merged if it does not pass the tests.
```

Comment:
Make this welcoming, like: `Please write tests also when you're adding a new feature.`